DEPENDENCIES

```diff
@@ -98,11 +98,31 @@ Prerequisites for building the OpenDataPlane (ODP) API
      --with-openssl-path=/home/${USER}/src/install-openssl-aarch64
    $ make
 
-3.3 Netmap packet I/O support (optional)
+3.3 ARMv8 Crypto native compile (optional)
+
+   Optimized crypto library using ARMv8 cryptographic extension for
+   AES-CBC, HMAC-SHA-1, HMAC-SHA-256 and AES-GCM algorithms.
+
+3.3.1 Building ARMv8 crypto library
+
+   # Checkout AArch64cryptolib code.
+   $ git clone https://github.com/ARM-software/AArch64cryptolib
+
+   Follow the build instructions in README.md
+
+3.3.1 Building ODP with ARMv8 crypto library
+   $ ./bootstrap
+   # Check the current package config path and append the armv8 crypto lib path
+   $ echo $PKG_CONFIG_PATH
+   $ export PKG_CONFIG_PATH=~AArch64cryptolib/pkgconfig
```

Comment:
`# Append the armv8 crypto lib path to PKG_CONFIG_PATH if necessary`
`$ export PKG_CONFIG_PATH=<AArch64cryptolib-path>/pkgconfig:$PKG_CONFIG_PATH`